_studio/shared/umc/codec/jpeg_dec/src/umc_mjpeg_mfx_decode.cpp

```diff
@@ -177,8 +177,8 @@ Status MJPEGVideoDecoderMFX::Close(void)
 
 void MJPEGVideoDecoderMFX::AdjustFrameSize(mfxSize & size)
 {
-    size.width  = mfx::align_value<uint32_t>(size.width, 16);
-    size.height = mfx::align_value<uint32_t>(size.height, m_interleaved ? 32 : 16);
+    size.width  = mfx::align2_value<uint32_t>(size.width, 16);
+    size.height = mfx::align2_value<uint32_t>(size.height, m_interleaved ? 32 : 16);
```

Comment:
Do we need exp. spec. here?

Reply:
I think - not

Reply:
Maybe I missed something but I don't see a change.

Reply:
I missed this one in my update commit